app/routes/events_.$eventId.tsx

```diff
@@ -26,6 +27,20 @@ export async function loader({ params }: LoaderFunctionArgs) {
   return json({ event });
 }
 
+export async function action({ params, request }: ActionFunctionArgs) {
+  const userSession = await requireUserSession(request);
+
+  if (!('userId' in userSession)) {
```

Comment:
use to get rid of typescript warning of userSession

Reply:
Just a nit, we could move this verification into `requireUserSession` with `tiny-invariant` or just this if-statement so we don;t have to do this everywhere we use `requireUserSession`.

Reply:
solved it by changing to throw redirect in requireUserSession